backend/src/main/java/gov/cdc/usds/simplereport/db/repository/DeviceTestPerformedLoincCodeRepository.java

```diff
@@ -0,0 +1,8 @@
+package gov.cdc.usds.simplereport.db.repository;
```

Comment:
for my own knowledge... this repository file is created 1:1 for models that perform CRUD operations for? so it was created for `DeviceTestPerformedLoincCode` but not for `CreateDeviceType` for example?